packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/template/ClientMethodTemplate.java

```diff
@@ -1254,12 +1254,22 @@ public static void generateJavadoc(ClientMethod clientMethod, JavaJavadocComment
         ProxyMethod restAPIMethod, boolean useFullClassName) {
         commentBlock.description(clientMethod.getDescription());
         List<ClientMethodParameter> methodParameters = clientMethod.getMethodInputParameters();
-        for (ClientMethodParameter parameter : methodParameters) {
-            commentBlock.param(parameter.getName(), parameterDescriptionOrDefault(parameter));
+        if (isPageStreamingClientMethodType(clientMethod.getType())) {
+            final MethodPageDetails methodPageDetails = clientMethod.getMethodPageDetails();
+            if (methodPageDetails != null) {
```

Comment:
The null check may not be necessary if `isPageStreamingClientMethodType` is true. Todo - verify this here and elsewhere, and remove the null check if it is safe to do so.

Reply:
verified and removed